applications/GeoMechanicsApplication/custom_constitutive/coulomb_with_tension_cut_off_impl.cpp

```diff
@@ -0,0 +1,163 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//
+//  Main authors:    Mohamed Nabi
+//                   Wijtze Pieter Kikstra
+//                   Anne van de Graaf
+//
+
+#include "custom_constitutive/coulomb_with_tension_cut_off_impl.h"
+#include "custom_utilities/constitutive_law_utilities.h"
+#include "geo_mechanics_application_variables.h"
+#include "includes/properties.h"
+#include "includes/serializer.h"
+
+namespace
+{
+
+using namespace Kratos;
+
+double CalculateApex(double FrictionAngle, double Cohesion)
```

Comment:
Just for for clarity:
```suggestion
double CalculateApex(double FrictionAngleInRadians, double Cohesion)
```